packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/CodeModelPlugin.cs

```diff
@@ -37,7 +38,7 @@ internal static CodeModelPlugin Instance
 
         public Configuration Configuration { get; }
 
-        public virtual IReadOnlyList<LibraryVisitor> Visitors => _visitors;
+        public IReadOnlyList<LibraryVisitor> Visitors => _visitors;
```

Comment:
Removed virtual as the same reasoning applies to visitors in that they should be additive.